cmd/crowdsec-cli/watchers.go

```diff
@@ -0,0 +1,152 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/crowdsecurity/crowdsec/pkg/database"
+	"github.com/enescakir/emoji"
+	"github.com/go-openapi/strfmt"
+	"github.com/olekukonko/tablewriter"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var machineID string
+var machinePassword string
+var machineIP string
+var interactive bool
+
+func NewWatchersCmd() *cobra.Command {
+	/* ---- DECISIONS COMMAND */
+	var cmdWatchers = &cobra.Command{
+		Use:   "watchers [action]",
+		Short: "Manage local API watchers",
+		Long: `
+Watchers Management.
+
+To list/add/delete watchers
+`,
+		Example: `cscli watchers [action]`,
+		Args:    cobra.MinimumNArgs(1),
+		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+			var err error
+			dbClient, err = database.NewClient(csConfig.DbConfig)
+			if err != nil {
+				log.Fatalf("unable to create new database client: %s", err)
+			}
+		},
+	}
+
+	var cmdWatchersList = &cobra.Command{
+		Use:     "list",
+		Short:   "List watchers",
+		Long:    `List `,
+		Example: `cscli watchers list`,
+		Args:    cobra.MaximumNArgs(1),
+		Run: func(cmd *cobra.Command, arg []string) {
+			watchers, err := dbClient.ListWatchers()
+			if err != nil {
+				log.Errorf("unable to list blockers: %s", err)
+			}
+			if csConfig.Cscli.Output == "human" {
+				table := tablewriter.NewWriter(os.Stdout)
+				table.SetCenterSeparator("")
+				table.SetColumnSeparator("")
+
+				table.SetHeaderAlignment(tablewriter.ALIGN_LEFT)
+				table.SetAlignment(tablewriter.ALIGN_LEFT)
+				table.SetHeader([]string{"Name", "IP Address", "Status"})
+				for _, w := range watchers {
+					var validated string
+					if w.IsValidated {
+						validated = fmt.Sprintf("%s", emoji.CheckMark)
+					} else {
+						validated = fmt.Sprintf("%s", emoji.Prohibited)
+					}
+					table.Append([]string{w.MachineId, w.IpAddress, validated})
+				}
+				table.Render()
+			} else if csConfig.Cscli.Output == "json" {
+				x, err := json.MarshalIndent(watchers, "", " ")
+				if err != nil {
+					log.Fatalf("failed to unmarshal")
+				}
+				fmt.Printf("%s", string(x))
+			} else if csConfig.Cscli.Output == "raw" {
+				for _, w := range watchers {
+					var validated string
+					if w.IsValidated {
+						validated = "true"
+					} else {
+						validated = "false"
+					}
+					fmt.Printf("%s %s %s\n", w.MachineId, w.IpAddress, validated)
```

Comment:
I think you want column names here. As for `ban list`, it's not consistent (" " vs ",")